packages/url_launcher/url_launcher/lib/url_launcher_string.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Provides a String-based alterantive to the Uri-based primary API.
+//
+// This is provided as a separate import because it's much easier to use
+// incorrectly, so should require explicit opt-in (to avoid issues such as
+// IDE auto-complete to the more error-prone APIs just by importing the
+// main API).
+
+export 'src/types.dart';
+export 'src/url_launcher_string.dart';
+export 'src/url_launcher_uri.dart';
```

Comment:
I don't think I'd export the URI API from this library. This file should only provide the "not recommended API".

(If users need both versions of the API they can still import both libraries: `url_launcher.dart` and `url_launcher_string.dart`)

Reply:
I did this because `closeInAppWebView` is only in `url_launcher_uri`. I guess if people only need that they could still import both though, so I'll remove it.